### PR TITLE
Use softmax as activation for the dense layer

### PR DESCRIPTION
**Pull request by Terahezi**
It is necessary to use softmax as activation to convert raw prediction values to probability distribution.

### CODE REVIEW

**Comment by Terahezi** [comment]
> https://cla.developers.google.com/

I signed it!

**Comment by bhanu-sharma** [comment]
> https://cla.developers.google.com/

I signed it!

**Comment by lamberta** [comment]
Can you also update the TF2 notebook: https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/load_data/images.ipynb

**Comment by yashk2810** [comment]
Also, can you update the loss in compile to use string losses?

Instead of `tf.keras.losses.sparse_categorical_crossentropy` use `'sparse_categorical_crossentropy'`

**Comment by Terahezi** [comment]
> Also, can you update the loss in compile to use string losses?
> 
> Instead of `tf.keras.losses.sparse_categorical_crossentropy` use `'sparse_categorical_crossentropy'`

But what is the diff?

**Comment by yashk2810** [comment]
There's no difference, its just a style preference 😀
